### PR TITLE
Support mapping time in the Store

### PR DESCRIPTION
**Pull request by daniel-thom**

### CODE REVIEW

File: src/chronify/store.py [hunk 1]
```diff
@@ -159,6 +179,12 @@ def ingest_from_csv(
             conn.commit()
         self.update_table_schema()
 
+    def map_time(self, src_schema: TableSchema, dst_schema: TableSchema) -> None:
```
Comment (lixiliu) [inline src/chronify/store.py:182]: Two versions of map time. I understand this one is a class method through store, would that be too confusing with the map_time() func?

File: tests/conftest.py [hunk 2]
```diff
@@ -12,10 +18,44 @@
...
 
 @pytest.fixture(params=ENGINES.keys())
 def iter_engines(request) -> Generator[Engine, None, None]:
+    """Return an iterable of sqlalchemy engines to test."""
     engine = ENGINES[request.param]
     yield create_engine(engine["url"], *engine["connect_args"], **engine["kwargs"])
+
+
+@pytest.fixture
+def one_week_per_month_by_hour_table() -> tuple[pd.DataFrame, int, TableSchema]:
```
Comment (lixiliu) [inline tests/conftest.py:33]: I was refactoring this out too in my stashed changes

**Review by lixiliu — APPROVED** [review]
A minor comment

File: src/chronify/store.py (hunk 1 above)
Comment (daniel-thom) [reply src/chronify/store.py:182]: Users will see the Store method. They will not see the backend method. This is a forwarding situation.  I would argue that consistency is better for developers in the code.

**Pull request merged**